.travis.yml

```diff
@@ -40,12 +40,11 @@ matrix:
     env: INSTALL_DEPENDS="pip==10.0.1 setuptools==30.4.0"
 
 before_install:
-  - python -m pip install --upgrade pip virtualenv
-  - virtualenv --python=python /tmp/venv
-  - source /tmp/venv/bin/activate
```

Comment:
I would prefer `virtualenv`s in general, because they are clean and implicit dependencies are harder to creep in. If we want to have a test entry that doesn't use them, to make sure that we upgrade existing test setups, that's fine, but I would make it conditional.

Reply:
Please note that you are already within a virtualenv in travis. We could use venv though.

Reply:
The problem is that it's a virtualenv that's precreated with various packages, such as `pytest`.

I've tried `venv`, but didn't have the patience to figure out how to use it. But I don't object to you figuring it out and letting me steal it for other projects. :-)

Reply:
> but didn't have the patience to figure out how to use it

Exactly the same as virtualenv :D

Reply:
Okay, I take it back - having a virtualenv imposed by Travis makes it updating virtualenv the only solution.

Reply:
Ah, so that's why it wasn't a drop-in replacement when I tried it. I guess you could try `deactivate`, but you may also find it not worth the time to fiddle with.